docs/resources/ces_alarmrule.md

```diff
@@ -11,57 +11,98 @@ Manages a Cloud Eye alarm rule resource within HuaweiCloud.
 ### Basic example
 
 ```hcl
-resource "huaweicloud_ces_alarmrule" "alarm_rule" {
-  alarm_name = "alarm_rule"
+variable "instance_id_1" {}
+variable "instance_id_2" {}
+variable "topic_urn" {}
+
+resource "huaweicloud_ces_alarmrule" "test" {
+  alarm_name           = "rule-test"
+  alarm_action_enabled = true
+  alarm_enabled        = true
+  alarm_type           = "MULTI_INSTANCE"
 
   metric {
-    namespace   = "SYS.ECS"
-    metric_name = "network_outgoing_bytes_rate_inband"
+    namespace = "SYS.ECS"
+  }
+
+  resources {
     dimensions {
       name  = "instance_id"
-      value = var.webserver_instance_id
+      value = var.instance_id_1
     }
   }
-  condition {
-    period              = 300
+
+  resources {
+    dimensions {
+      name  = "instance_id"
+      value = var.instance_id_2
+    }
+  }
+
+  condition  {
+    period              = 1200
     filter              = "average"
     comparison_operator = ">"
-    value               = 6
+    value               = 6.5
+    unit                = "B/s"
+    count               = 1
+    suppress_duration   = 300
+    metric_name         = "network_outgoing_bytes_rate_inband"
```

Comment:
Does the `network_outgoing_bytes_rate_inband` is the fixed value? If not, it might be better to use variables?

Reply:
it's a fixed value